controller/environment.go

```diff
@@ -132,3 +143,17 @@ func (c *EnvironmentController) Show(ctx *app.ShowEnvironmentContext) error {
 	}
 	return ctx.OK(res)
 }
+
+func (c *EnvironmentController) checkClustersUser(ctx context.Context, clusterURL string) error {
```

Comment:
could you add a test for this func?

Reply:
this [link](https://github.com/fabric8-services/fabric8-env/pull/22/files#diff-5d0c590540f7c2ddcb9dd555d0d7d339R83) test is added to test this function only.  If you observe blackbox_tests, it is a way to test all different combination.  This new test is added such that only `checkClustersUser` function is tested rest are assumed to be working.  Do you still see need to add test, which will have less pre-setup but everything else same as from part of blackbox test ??